### ISSUE TITLE
update or delete create-react-context

### ISSUE THREAD

**Issue by dartess** [gribnoysup/react-yandex-maps#238]
Hello!

Share why you use `create-react-context` project?

If this is aimed at supporting very old versions of React and you cannot or do not want to refuse to support them, can you at least upgrade `create-react-context` package to `0.3`? At the moment, this forces the very old version of `core-js` to be installed in the project.

```
> npm ls core-js
├─┬ react-yandex-maps@4.4.0
│ └─┬ create-react-context@0.2.3
│   └─┬ fbjs@0.8.17
│     └── core-js@1.2.7
```

`create-react-context@0.3.0` don't has `fbjs` as deps and this problem will be eliminated. Removing this dependency, however, will also help :)

**Comment by mmarkelov** [comment]
@dartess as I can remember `create-react-context` was used for supporting old React versions. As for me it's nice to remove it, cause official context API was introduced about 2 years ago. @gribnoysup what do you think?

**Comment by gribnoysup** [comment]
Removing it completely will be a breaking change that I don't want to do at the moment as I'm in the process of working on a library refactor. Updating a dependency is a good idea that I'll be happy to release 👍

**Comment by gribnoysup** [comment]
Released as `4.5.0`
